### PR TITLE
Disable next config headers

### PR DESCRIPTION
**Pull request by cb-jake**
### _Summary_

Remove the next.config headers for the SDK playground

### _How did you test your changes?_

Manually. SDK playground should load normally

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: examples/testapp/next.config.js [hunk 1]
```diff
@@ -1,18 +1,4 @@
...
  */
-module.exports = {
-  async headers() {
-    return [
-      {
-        source: "/",
-        headers: [
-          {
-            key: "Cross-Origin-Opener-Policy",
-            value: "unsafe-none", // Or 'same-origin' to test COOP errors
```
Comment (spencerstock) [inline examples/testapp/next.config.js:12]: Could this still be a valuable example? Maybe we move it instead of removing it?

Comment (cb-jake) [reply examples/testapp/next.config.js:12]: Yeah I think its valuable for testing. Let me add a new PR that adds a headers file that users can import when they want to test coop errs

Comment (cb-jake) [reply examples/testapp/next.config.js:12]: added here [6a2c8b8](https://github.com/coinbase/coinbase-wallet-sdk/pull/1418/commits/6a2c8b83c2e2d5e77a39b2c14be249713465b0eb)

File: examples/testapp/tsconfig.json [hunk 2]
```diff
@@ -1,10 +1,6 @@
 {
   "compilerOptions": {
-    "lib": [
```
Comment (cb-jake) [inline examples/testapp/tsconfig.json:3]: No changes just formatted

**Review by spencerstock — APPROVED** [review]
Very nice.

**Pull request merged**